swagger-ui.version

```diff
@@ -1 +1 @@
-v4.15.5
+v5.0.0-alpha.4
```

Comment:
I suggest to wait for a stable version before releasing this.
It's not a critical component but it could still have some unexpected issues blocking integration with other tools (Lago documentation, Postman...)